src/sentry/api/serializers/models/team.py

```diff
@@ -187,20 +197,43 @@ def serialize(
         return d
 
 
+def get_scim_teams_members(
+    team_list: Sequence[Team],
+) -> MutableMapping[Team, MutableSequence[MutableMapping[str, Any]]]:
+    members = RangeQuerySetWrapper(
+        OrganizationMember.objects.filter(teams__in=team_list).select_related("user"),
+        limit=10000,
+    )
+    member_map: MutableMapping[Team, MutableSequence[MutableMapping[str, Any]]] = defaultdict(list)
```

Comment:
@getsentry/python-typing if i don't include this verbose type here i get the error:
```
error: Incompatible return value type (got "defaultdict[Any, List[Dict[str, Any]]]", expected "MutableMapping[Any, MutableSequence[MutableMapping[str, Any]]]")
```
which doesn't make sense to me since defaultdict should be a mutable mapping? anyone have ideas?